### ISSUE TITLE
False positive of blob container access

### ISSUE THREAD

**Issue by BernieWhite** [Azure/PSRule.Rules.Azure#1212]
**Description of the issue**

In the case where `properties.publicAccess` for a blob container is not specified a fail occurs.

**To Reproduce**

Consider:

```bicep
resource container 'Microsoft.Storage/storageAccounts/blobServices/containers@2021-06-01' = {
  name: 'account/default/${name}'
}
```

**Expected behaviour**

The default if not specified of `properties.publicAccess` is `None` or private, so if the property is not defined it is not an issue.

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **1.11.1**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      7.2.1
PSEdition                      Core
GitCommitId                    7.2.1
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
